src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.csproj

```diff
@@ -63,6 +63,9 @@
     <Compile Include="System\Diagnostics\SymbolStore\ISymbolDocumentWriter.cs" />
   </ItemGroup>
   <ItemGroup>
+    <!-- Some of the P2P references this csproj has are building for netstandard, so need to add reference to
+    netstandard shim -->
+    <Reference Include="netstandard" />
```

Comment:
This one is likely going to need to be fixed to reference a netcoreapp version of MetadataReader.